packages/ngworker/lumberjack/src/lib/configuration/lumberjack-root.module.ts

```diff
@@ -42,7 +46,11 @@ export function logDriverConfigFactory({ levels }: LumberjackConfig): Omit<Lumbe
       provide: lumberjackLogDriverConfigToken,
       useFactory: logDriverConfigFactory,
     },
-    { provide: LumberjackService, useClass: LumberjackService },
+    LumberjackService,
+    LumberjackLogFormatter,
+    LumberjackLogDriverLogger,
+    LumberjackLogFactory,
+    LumberjackTimeService,
```

Comment:
nit: Can we list these providers in alphabetical order?
```suggestion
    LumberjackLogDriverLogger,
    LumberjackLogFactory,
    LumberjackLogFormatter,
    LumberjackService,
    LumberjackTimeService,
 ```